packages/contracts/contracts/libs/v0.8.x/minter-libs/MaxInvocationsLib.sol

```diff
@@ -250,27 +250,11 @@ library MaxInvocationsLib {
         uint256 projectId,
```

Comment:
should this be named `invocationsRemainOnCore`?

Reply:
Good call, `invocationsRemainOnCore` is clearer.. Updated this + the call in SEALib